activerecord/lib/active_record/relation/batches.rb

```diff
@@ -28,10 +28,11 @@ module Batches
     # ==== Options
     # * <tt>:batch_size</tt> - Specifies the size of the batch. Default to 1000.
     # * <tt>:start</tt> - Specifies the primary key value to start from.
+    # * <tt>:end_at</tt> - Specifies the ending point for the batch processing.
```

Comment:
Actually, I'd be :-1: on having a range option. We can't really change the start option. So the end option is a nice compliment, whereas the range plus start-end seems excessive.